tests/integration/page_objects/PageFactory.py

```diff
@@ -29,16 +28,20 @@ def get_page_object(page_name,base_url=conf.base_url_conf.base_url):
         elif page_name in ["login","sso"]:
             test_obj = Login_Page(base_url=base_url)
         elif page_name == "add articles page":
-            test_obj =  Addarticles_Page(base_url=base_url)  
+            test_obj =  Addarticles_Page(base_url=base_url)
         elif page_name == "manage articles page":
             test_obj = Managearticles_Page(base_url=base_url)  
         elif page_name == "edit articles page":
-            test_obj = Editarticles_Page(base_url=base_url)  
+            test_obj = Editarticles_Page(base_url=base_url)
         elif page_name == "create newsletter page":
             test_obj = Createnewsletter_Page(base_url=base_url)
         elif page_name == "preview newsletter page":
-            test_obj = Previewnewsletter_Page(base_url=base_url)    
+            test_obj = Previewnewsletter_Page(base_url=base_url)            
```

Comment:
Can you add a comment here at the end saying any new page added needs to be updated in the `get_all_page_names` method too.